packages/truffle-db/src/artifacts/json/resolvers.ts

```diff
@@ -32,6 +36,8 @@ export const resolvers = {
             json: JSON.stringify(artifact.ast)
           },
           source: { contents: artifact.source, sourcePath: artifact.sourcePath },
+          bytecode: { bytes: linkedBytecodeCreate.bytes, linkReferences: linkedBytecodeCreate.linkReferences },
```

Comment:
Should just be able to do:
```suggestion
          bytecode: shimBytecode(artifact.bytecode),
```

(and same for deployed)